modules/oadp-using-data-mover-for-csi-snapshots.adoc

```diff
@@ -84,18 +116,47 @@ spec:
         default: true
         objectStorage:
           bucket: <bucket_name>
-          prefix: <bucket_prefix>
+          prefix: <bucket-prefix>
         provider: aws
   configuration:
     restic:
       enable: <true_or_false>
     velero:
+       itemOperationSyncFrequency: "10s"
       defaultPlugins:
         - openshift
         - aws
         - csi
+        - vsm <1>
+  features:
+    dataMover:
+      credentialName: restic-secret
+      enable: true
+      maxConcurrentBackupVolumes: "3" <2>
+      maxConcurrentRestoreVolumes: "3" <3>
+      pruneInterval: "14" <4>
+      volumeOptions: <5>
+        sourceVolumeOptions:
+          accessMode: ReadOnlyMany
+          cacheAccessMode: ReadWriteOnce
+          cacheCapacity: 2Gi
+        destinationVolumeOptions:
+          storageClass: other-storageclass-name
+          cacheAccessMode: ReadWriteMany
+  snapshotLocations:
+    - velero:
+        config:
+          profile: default
+          region: us-west-2
+        provider: aws
+
 ----
-<1> Add the Restic secret name from the previous step. If this is not done, the default secret name `dm-credential` is used.
+<1> OADP 1.2.x only.
```

Comment:
Do you need to define what <1> is?

Reply:
No. It says in the DPA that it is a defaultplugin.